OpenSim/Common/Reporter.h

```diff
@@ -113,6 +113,19 @@ class Reporter : public AbstractReporter {
     OpenSim_DECLARE_LIST_INPUT(inputs, InputT, SimTK::Stage::Report,
         "Variable list of quantities to be reported.");
 
+    //=============================================================================
+    // PUBLIC METHODS
+    //=============================================================================
+
+    // Allow overloading of updInput
+    using AbstractReporter::updInput;
+
+    /** Convenience method that can be used in place of updInput("inputs"). **/
```

Comment:
This description is clear. Consider adding in a code snippet example:

```
...
@code
<example>
@endcode
```

Also, could you make `updInput("inputs")` appear as code by surrounding it with backticks?